core/controllers/resources.py

```diff
@@ -56,7 +56,7 @@ class AssetDevHandler(base.BaseHandler):
     _SUPPORTED_PAGE_CONTEXTS = [
         feconf.ENTITY_TYPE_EXPLORATION, feconf.ENTITY_TYPE_SKILL,
         feconf.ENTITY_TYPE_TOPIC, feconf.ENTITY_TYPE_STORY,
-        feconf.ENTITY_TYPE_QUESTION
+        feconf.ENTITY_TYPE_QUESTION, feconf.ENTITY_TYPE_SUGGESTION
```

Comment:
Hi @donosco98 -- sorry about this, but let's not introduce "suggestion" as an entity type. It's quite confusing compared to the others.

I had a quick chat with @DubeySandeep and we are thinking about using /exploration-suggestions/{{exp-id}} and /question-suggestions/{{question-id}} instead. He'll follow up with you, if that's OK!

Reply:
done